README.md

```diff
@@ -0,0 +1,8 @@
+This repository contains the cray-nexus helm chart. The chart creates the sonatype nexus pod and sets the pod up for use.
+
+
+## Image Upgrade Procedure
+
+To update the image version that nexus uses a new version must be made in the (container images repo)[https://github.com/Cray-HPE/container-images/tree/main/docker.io/sonatype/nexus3]. To make a new version everything from the latest previous version (3.38.0 - 04/2022) should be copied over to the new version. To create a new version there is a script that can be run from the container-images repository. This will set up the folder and docker file, to ensure nexus is created correctly there is two files that must be brought into the new folder for the new version. The create-trust-store.sh and nexus3-keycloak-plugin-0.5.0-bundle.kar. Then the docker file must be edited to reflect how it is created in the latest previous version (3.38.0 - 04/2022). Once the files have been created and edited a pull request should be made to push the new folder and workflow to main. The pull request will run the workflow and tests then upload the container image to (artifactory)[https://artifactory.algol60.net/ui/repos/tree/General/csm-docker%2Fstable%2Fdocker.io%2Fsonatype%2Fnexus3] to be used in the cray-nexus chart.
+
```

Comment:
I like the level of detail.  I might suggest that you create an upgrade readme in the container images nexus3 project (with the detail of what to copy, etc) and just refer the developer to that readme for the detail from your cray-nexus chart repo "Image Upgrade Procedure" readme.  That way you or another dev is not forced to update the instructions of the nexus3 image upgrade process in the chart repo (for example if a file is added to the image, etc).  Nexus3 image specific changes and associated readme can just be updated in a the same repo.

The chart aspect of the readme seems fine although IMO I'd try to eliminate the line numbers (if possible) as those may shift over time and may not get updated.